webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
- [ ] Let make sure the usage of `import` and `export` instead of `require` or `module.exports`